app/CocoaPods/CPHomeWindowController.m

```diff
@@ -55,6 +55,9 @@ - (void)windowDidLoad;
 - (void)didDoubleTapOnRecentItem:(NSTableView *)sender;
 {
   NSInteger row = [sender selectedRow];
+  if (row < 0) {
```

Comment:
`[tableView selectedRow]` returns -1 when there is no row selected. On empty table you can still double click and before this check there is a silent crash.